code/game/machinery/requests_console.dm

```diff
@@ -497,8 +600,26 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
 			var/obj/item/weapon/stamp/T = O
 			msgStamped = text("<font color='blue'><b>Stamped with the [T.name]</b></font>")
 			updateUsrDialog()
+	if (istype(O, /obj/item/telephone) && landline)
+		landline.attackby(O, user)
 	return
 
+/obj/machinery/requests_console/verb/pick_up_phone()
+	set category = "Object"
+	set name = "Pick up telephone"
+	set src in oview(1)
```

Comment:
probably no difference but view would be more appropriate than oview in this context

Reply:
more appropriate why?
from what i've seen they're used basically interchangably, is there a meaningful difference?
oview should be slightly faster since it doesn't check the contents if the DM reference is to be believed

Reply:
oview excludes the tile of origin, so for example if you were standing on the same tile as the RC console

Reply:
i did some reading and still don't quite understand but tested ingame it works on the same tile 
only time oview doesn't work is when you `forceMove` the RC into yourself(works when you `forceMove` yourself into the RC, though)

Reply:
Yeah, that's why I said in the first comment that it probably makes no difference given how verbs populate but that view() is what you're actually trying to do. I don't think either one is faster in any meaningful way, but I do think view() is what you actually mean here. This is a pretty low priority change but I still would advise it.